app/src/test/java/nl/tudelft/b_b_w/model/TrustValuesUnitTest.java

```diff
@@ -8,6 +8,15 @@
 public class TrustValuesUnitTest {
 
     /**
+     * testTrustValuesInitialized
+     * tests whether the trust value of the initalized is correct
+     */
+    @Test
+    public void testTrustValuesInitialized() {
+        assertEquals(20, TrustValues.INITIALIZED.getValue());
```

Comment:
Testing enum values is indeed unnecessary. However, we still need to implement this test to get the line coverage.